Add article sections clarifying tournament bans and their reach (#6323)
ppy/osu-wiki: wiki/Help_Centre/Account/en.md

Refer to *[Account Restrictions](/wiki/Help_Centre/Account_Restrictions)* for more information on account restrictions and the appeal process.

## Tournament bans

### What is a tournament ban?

Tournament bans are as the name describes - a ban from participation in all officially supported tournaments. This includes playing, staffing or assisting a tournament in any way.

> A user may receive a tournament ban for deliberately breaking the osu! community rules during tournament play, depending on the severity of the offense. Examples of such bad behaviour include attempting to gain an unfair advantage in an officially supported tournament by any means, being abusive to anyone involved in a tournament, being deliberately disruptive to the running of a tournament, and more.

Reviewer: ```suggestion
A user may receive a tournament ban for deliberately breaking the osu! community rules during tournament play, depending on the severity of the offense. Examples of such bad behaviour include attempting to gain an unfair advantage in an officially supported tournament by any means or being abusive to anyone involved in a tournament, being deliberately disruptive to the running of a tournament.
```
